examples/smqtk/show_smqtk.py

```diff
@@ -0,0 +1,23 @@
+"""Meta-configuration demonstration."""
+
+from tinker import protocol
+from smqtk_core import Configurable
+
+
+class ShowSmqtk(protocol.Protocol):
+    """
+    A protocol designed for testing SMQTK integration.
+
+    Will initialize and then print out config / instance details
+    """
+
+    def get_config(self):
```

Comment:
Should python files under the example space also be subject to mypy type checking?

Reply:
Maybe. Since these are examples, I don't think it's as directly useful to force ourselves to supply type annotations for these. On the other hand, promoting type annotation in Python projects in general sounds like a good idea. I'm open to suggestions here.

Reply:
I was thinking along the lines of promoting type annotations as well as being a method of checking that we do not unwittingly create incorrect examples.

Reply:
See #142.